tstest/integration/vms/vms_test.go

```diff
@@ -80,6 +86,40 @@ func (d *Distro) InstallPre() string {
 	return ""
 }
 
+// fetchFromS3 fetches a distribution image from Amazon S3 or returns false if it
+// lacks permission. In that case it will fall back to downloading the distro from
+// the public internet.
+func fetchFromS3(t *testing.T, fout *os.File, d Distro) bool {
```

Comment:
you don't document how fout is used. Is it closed?

Reply:
91bdf46a documents how fout is consumed.